src/components/line/filterToolbox/chartLabels.js

```diff
@@ -120,7 +120,9 @@ const ChartLabels = ({ labelProps, ...rest }) => {
 
   const onLabelValueClick = ({ label, value }) => {
     const labelsToSet = { ...selectedLabels }
-    const shouldAddLabelValue = !labelsToSet[label]?.includes?.(value)
+
+    const labelValues = labelsToSet[label]?.split?.("|") ?? []
+    const shouldAddLabelValue = !labelValues.includes?.(value)
```

Comment:
Not sure if it has any value to add these on a util function!
I see that the code is repeated here!